docs/reference/publishers/aws.md

```diff
@@ -0,0 +1,22 @@
+# AWS S3
+A publisher to upload composed experiences to Atama's private AWS S3 bucket.
```

Comment:
Can we rename this to "Delivery API Publisher"? The fact it's a S3 bucket is an implementation detail that may change in the future.